README.md

```diff
@@ -63,3 +63,58 @@ Google Sign-In also supports iOS apps that are built for macOS via
 [Mac Catalyst](https://developer.apple.com/mac-catalyst/).  In order for your Mac Catalyst app
 to store credientials via the Keychain on macOS, you will need to
 [sign your app](https://developer.apple.com/support/code-signing/).
+
+## Using the Google Sign-In Button
+
+Depending upon your platform, GoogleSignIn-iOS provides a few options to use
+when creating a sign-in button.
+
+### SwiftUI (iOS and macOS)
+
+Creating a Google sign-in button in SwiftUI can be as simple as this:
+
+```
+GoogleSignInButton {
+  GIDSignIn.sharedInstance.signIn(
+    with: configuration, 
+    presenting: yourViewController) { user, error in
+      // check `error`; do something with `user`
+  }
+}
+```
+
+This example takes advantage of the initializer's [default argument for the
+view model](GoogleSignInSwift/Sources/GoogleSignInButton.swift#L39).
+The default arguments for the view model will use the light scheme, the
+standard button style, and the normal button state.
+You can supply an instance of [`GoogleSignInButtonViewModel`](GoogleSignInSwift/Sources/GoogleSignInButtonViewModel.swift)
+with different values for these properties to customize the button.
+[This convenience initializer](GoogleSignInSwift/Sources/GoogleSignInButton.swift#L56) 
+provides parameters that you can use to set these values as needed.
+
+### UIKit (iOS)
+
+If you are not using SwiftUI to build your user interfaces, you can either
+create `GIDSignInButton` programmatically, or in a Xib/Storyboard. 
+If you are writing programmatic UI code, it will look something like this:
+
+`GIDSignInButton *button = [[GIDSignInButton alloc] initWithFrame: CGRectMake(<YOUR_RECT>)];`
```

Comment:
Should we provide this code sample in Swift to match the other two?

Reply:
Sure thing